GenderPayGap.WebUI/BackgroundJobs/ScheduledJobs/SendReminderEmailsJob.cs

```diff
@@ -68,12 +67,16 @@ private void SendReminderEmailsForSectorType(SectorTypes sector, string runId, D
                         .Where(user => !user.HasBeenAnonymised)
```

Comment:
I know this isn't part of this ticket, but would it be better if we break down the SendReminderEmailsForSectorType into multiple functions?
Something like:
```
`SendReminderEmailsForSectorType() {
    foreach(year) {
        if (IsAfterEarliestReminderforReportingYear()) {
            sendReminderEmailsForSectorTypeAndYear()
        }   
    }
}

sendReminderEmailForSectorTypeAndYear() {
    foreach(user) {
        stopIfProcessRunningLongerThan(45)
        sendReminderEmail()
    }
}
`
```